api/src/main/java/org/openmrs/module/openconceptlab/importer/Saver.java

```diff
@@ -362,6 +362,9 @@ public Item saveMapping(final CacheService cacheService, final Import update, fi
 					if (oclMapping.getMapType().equals(MapType.Q_AND_A)) {
 						item = updateOrAddAnswersFromOcl(update, oclMapping, fromConcept, toConcept);
 					} else {
+						if (!fromConcept.getSet()) {
```

Comment:
Thanks @jwamalwa ,well i think we have could follow syntax of if else ie ( if {  

}
else if {

}

else {

}